src/components/SearchBar/__tests__/SearchBar.spec.js

```diff
@@ -73,7 +73,6 @@ describe('SearchBar', () => {
   });
 
   it('executes the search function when the user types', async () => {
-    jest.useFakeTimers();
```

Comment:
Neither the test nor the component were using any timers (functions like setTimeout or setInterval) so I'm pretty sure this is unnecessary.